packages/tx-construction/src/tx-builder/OutputBuilder.ts

```diff
@@ -21,6 +21,8 @@ export interface OutputBuilderProps {
   txOut?: PartialTxOut;
   /** Logger */
   logger: Logger;
+  /** Handle Provider for ADAHandle */
```

Comment:
```suggestion
  /** Handle Provider for resolving an address from the supplied handle */
```
Let's avoid using the ADA Handle brand, since we're implementing the _Handle Standard_